macos/create_installer_macos.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+command -v gfind || ( echo "Install gfind (eg. sudo port install findutils)" ; exit 1)
+
+rm -rf macos_installer/
+mkdir macos_installer/
+cd macos_installer/
+
+COQ_VERSION=$(coqc --print-version | cut -d ' ' -f 1 | tr -d '\r')
+APP=_dmg/Coq_${COQ_VERSION}.app/
+
+opam source coq --dir=coq/
+eval `(cd coq ; ./configure -local >/dev/null 2>&1 ; grep VERSION4MACOS config/Makefile)`
```

Comment:
@gares: I picked this up now. Can you explain why extracting VERSION4MACOS from the configure result is required and one can't simply use COQ_VERSION? Does this have to do with the naming of beta versions? Is the usage of COQ_VERSION above then consistent?

Reply:
I think apple needs x.y.z, see the comments in configure.ml

Reply:
That's what I thought. But I think calling configure for this is a bit heavy - I will pull it from the configure.ml file instead.

Reply:
Thinking about it: shouldn't this be the platform version?

Reply:
I'm fine with that.
The only problem is the beta, IIRC we use 8.12.91 to mean 8.13+beta1 or something like that, which is not exactly nice.